libkbfs/interfaces.go

```diff
@@ -942,12 +942,6 @@ type BlockOps interface {
 	Ready(ctx context.Context, kmd KeyMetadata, block Block) (
 		id BlockID, plainSize int, readyBlockData ReadyBlockData, err error)
 
-	// Put stores the readied block data under the given block
-	// pointer (which belongs to the TLF with the given ID) on the
-	// server.
-	Put(ctx context.Context, tlfID TlfID, blockPtr BlockPointer,
```

Comment:
If we end up making quota reclamation talk to the real block server directly, we might need to add this back, or have some other way of distinguishing talking to the 'real' block server vs. the journal block server. But we can figure that out when we get there...